server/public/layout.css

```diff
@@ -0,0 +1,16 @@
+#root {
+  background-color: #c4c4c4;
+  height: 100vh;
+}
+
+.height-500 {
+  height: 500px;
+}
+
+.text-align-center {
+  text-align: center;
+}
+
+.padding {
+  padding: 4rem;
+}
```

Comment:
I recommend adding a `.container` class that you can set a max-width on and `margin: 0 auto;` so that your main page content centers on the page like in your figma.